examples/waspleau/.gitignore

```diff
@@ -0,0 +1,3 @@
+/.wasp/
+/.env
+.DS_Store
```

Comment:
did you have to add this manually or was it already set here by Wasp? I ask b/c I don't remember adding it, and Martin is on Linux :D so we make sure it is added by default if it isn't atm

Reply:
When running `wasp new` we automatically get a `.gitignore` file with the following:
```
/.wasp/
/.env
```

Should I open an issue to add more things to the default `.gitignore` like `.DS_Store`?

Reply:
.DS_Store is mac specific right? So maybe lets leave it out for now, and we will see, if it turns out people need it we can put it in.

Reply:
Yeah, that is Mac specific. Sounds good. I was going to open a beginner issue with a request for more files and patterns to add to `.gitignore` and `.waspignore` (both platform specific and perhaps editor based). Still worthwhile to do so to get some new contributors?

I was going to encourage them to look at: https://github.com/github/gitignore for inspiration